### ISSUE TITLE
Functions.sk

### ISSUE THREAD

**Issue by Blobbyguy** [SkriptLang/Skript#1814]
### Description

Seen as skript loads directories first, the only way to ensure your functions.sk file loads first (to avoid errors) is to put it in its own directory and have it load it first.

My suggestion is for skript to have dedicated functions.sk file that always loads first before any of the directories or files.

**Comment by ShaneBeee** [comment]
You can just rename it.
a-functions.sk, or if you have directions, name the direction A, and put your functions in there.

In my opinion, I really don't see the point of this. People may call their "functions" skript other things, like API

**Comment by TheBentoBox** [comment]
I agree that this seems unnecessary. I don't think Skript should be silently imposing functionality based on file names. There's nothing wrong with having to use sub-directories or naming patterns to define load order, and it's also much more clear than a random file from further down in the file list being loaded first just because that's how it works.

**Comment by Nicofisi** [comment]
The changelog for 2.3 says:
> Added ability to place function calls before function definitions

I assumed that worked with multiple files too, I understand it's not the case?

**Comment by bensku** [comment]
It does work with multiple files, but you can't call functions that have not been yet loaded from script load events.
